src/main/java/net/sf/jabref/logic/importer/MrDLibFetcher.java

```diff
@@ -87,6 +88,7 @@ private String makeServerRequest() {
         try (CloseableHttpClient httpclient = HttpClients.custom().setSSLSocketFactory(sslsf).build()) {
         Unirest.setHttpClient(httpclient);
 
+            String query = selectedEntry.getField("title").get().replaceAll("\\{|\\}", "");
```

Comment:
Haven't looked at the code in depth, so just a quick comment here: Instead of removing the braces manually you should use the `getLatexFreeField("title")` method which removes braces and also converts some LaTeX functions correctly which might be broken by your approach (e.g., `\textsuperscript{2}`).